docs/admin/troubleshooting.md

```diff
@@ -921,7 +921,8 @@ pool header correct
 
 ## Syslog
 
-RAS events are printed to the Syslog by 'daos_server' processes via the golang standard library API.
+[`RAS events`](https://docs.daos.io/v2.2/admin/administration/#ras-events) are printed to the Syslog
```

Comment:
Thanks for adding the link. My only concern with this is that it should probably be a relative link so that it stays current as the documentation version changes, right?

Reply:
I checked and it seems like this is the chosen way to link, there are no relative links in the admin guide between different files. I think it's best to conform with regard to the format, does that sound okay?

Reply:
Seems strange to me that the other links aren't relative, but if that's the convention then I agree that we should follow it.

Reply:
Wondering, are these links auto-updated for versioning changes? I noticed we're pointing into a specific version (2.2) here.

Reply:
yes we are, that's the same across the admin guide so I assume it's part of a process